scss/components/_case-card.scss

```diff
@@ -0,0 +1,97 @@
+.civicase__case-card {
+  box-shadow: none;
+
+  .panel-body {
+    background-color: transparent;
+    border: 0 !important;
+  }
+
+  .civicase__contact-card {
+    font-size: 14px;
+    font-weight: 600;
+    line-height: 18px;
+  }
+
+  .civicase__contact-icon {
+    color: $crm-grayblue-darker;
+    font-size: 24px;
+    margin-top: -3px;
+  }
+
+  .civicase__checkbox {
+    left: 20px;
+    top: 15px;
+  }
+}
+
+.civicase__case-card--closed {
+  background-image: repeating-linear-gradient(60deg, $gray-light, $gray-light 2px, $gray-lighter 2px, $gray-lighter 20px);
+  min-height: 150px;
+
+  .civicase__contact-name,
+  .civicase__contact-additional__container,
+  .civicase__case-card__type {
+    text-decoration: line-through;
+  }
+
+  .civicase__case-card__next-milestone {
+    display: none;
```

Comment:
This should not controlled from the css. For closed cases, there are no next activities thats why we dont see them. So we should do it part of the JS functionality.